tests/test-pull-corruption.sh

```diff
@@ -55,7 +55,16 @@ do_corrupt_pull_test() {
 
 # FIXME - ignore errors here since gjs in RHEL7 has the final
 # unrooting bug
-gjs $(dirname $0)/corrupt-repo-ref.js ${repopath} main || true
+
+# super hacky fix to support both installed and uninstalled tests. we'll
+# probably want to just enable it for one of them.
+if [ -f /lib64/libostree-1.so ]; then
```

Comment:
Yeah, agreed @smcv's fix here is better.  Jonathan, what do you think about rebasing your patch series on top of his?

I think now that we have the PR tester enabled we're going to have to merge to one PR.  @smcv maybe do any `fixup!`s on your PR, then Jonathan pulls it into this one?